### DISCUSSION TITLE
冰！！！

### DISCUSSION THREAD

**Discussion by 2601942509** [share121/inter-knot#307] [其他]
![1722873778461](https://github.com/user-attachments/assets/625d8cf5-57df-44d4-b960-3d3c4e00611e)
我家小孩！不懂事去隔壁方舟玩了！

**Comment by lappslover** [comment]
？你怎么出来了快跟我回去（懒得复制了各位自行搜索佩佩ep

**Reply by paul-way52** [reply]
诶？你怎么出来了，肘，跟我回去🤓👇https://www.bilibili.com/video/BV1Sf421q7dN

**Comment by Acter064** [comment]
我焯，冰！！！肘，跟我一起去溜冰！https://www.bilibili.com/video/BV1Ky411q7QC/?spm_id_from=333.788.recommend_more_video.1&vd_source=72b0d39598129a180e8a4efac364c3bb

**Comment by paul-way52** [comment]
呱！不要口牙😱我才刚溜完口牙😭

**Comment by Alorit** [comment]
掺冰冰！掺冰冰！

**Comment by kano0x0** [comment]
爱丽丝不想再溜冰了  :(

**Comment by ACGNworld** [comment]
谁家小孩！
